src/main/java/cubicchunks/util/MathUtil.java

```diff
@@ -112,6 +112,26 @@ public static double gaussianProbabilityDensity(double x, double mean, double st
         return exp(-(x - mean) * (x - mean) / (2 * stdDev * stdDev)) /
                 (sqrt(2 * Math.PI) * stdDev);
     }
+	
+    public static double gaussianProbabilityCyclic(int x, int mean, double stdDev, int spacing) {
+        /* Modulo works from 0 to limit-1.
```

Comment:
I suggest to use `/** **/` instead of `/* */` and write comment right above function. Also avoid writing what you want and write what function do instead.